tests/Resources/Fixtures/config/full.xml

```diff
@@ -62,6 +62,7 @@
             </authentication>
             <cache cache-pool="my_cache_pool" stream-factory="my_other_stream_factory">
                 <config default-ttl="42" cache-lifetime="2592000" hash-algo="sha1" cache-key-generator="null">
+                    <cache-listeners></cache-listeners>
```

Comment:
afaik this should be <cache-listener>, symfony configuration handles pluralization for xml. but lets do the full test with 2 listeners instead of 0.